src/fluidsynth.c

```diff
@@ -403,11 +356,40 @@ int main(int argc, char **argv)
     int fast_render = 0;
     static const char optchars[] = "a:C:c:dE:f:F:G:g:hijK:L:lm:nO:o:p:QqR:r:sT:Vvz:";
 
-#ifdef _WIN32
+#if defined(_WIN32)
+    char **argv = NULL;
     // console output will be utf-8
     SetConsoleOutputCP(CP_UTF8);
     // console input, too
     SetConsoleCP(CP_UTF8);
+    // conversion of wchar_t (UTF-16) arguments to char (UTF-8)
+    if ((argv = (char **) calloc( argc, sizeof(char *) )) == NULL)
+    {
+        fprintf(stderr, "Out of memory\n");
+        goto cleanup;
+    }
+    else
+    {
+        for (i = 0; i < argc; ++i)
+        {
+            int u8_count = 0;
+            if (1 > (u8_count = WideCharToMultiByte(CP_UTF8, WC_ERR_INVALID_CHARS, wargv[i], -1, NULL, 0, NULL, NULL)))
```

Comment:
@pedrolcl We have a minor oversight: `WC_ERR_INVALID_CHARS` is not available for WindowsXP, causing the CI to fail:
https://dev.azure.com/tommbrt/tommbrt/_build/results?buildId=10875&view=logs&j=ae601b3d-8756-52a4-4bcc-59c4d24c3f8c&t=1e42d489-b2c6-5e77-903e-6d797975357c&l=653

JJC needs WindowsXP compatibility. I haven't seen this earlier, because the build failed due to an issue in libinstpatch related to your bump of libsndfile min version: https://github.com/swami/libinstpatch/pull/78

Reply:
Then, the only option is to replace the WC_ERR_INVALID_CHARS flag by 0.

Reply:
Would something like

```c
#ifndef WC_ERR_INVALID_CHARS
#define WC_ERR_INVALID_CHARS 0
#endif
```

make sense, so that at least on modern Windows it reports a proper error?